app/CustomFields/HeaderDefinition.php

```diff
@@ -37,6 +38,44 @@ public static function get($post_type) {
           ),
         ),
         array (
+          'key' => $key . '_video_file',
+          'label' => 'Background video',
+          'name' => 'background_video',
```

Comment:
This `name` should match `key`

`'key' => $key . '_video_file'`
`'name' => 'video_file',`

Or rename them both to `background_video`